scripts/test/ci_tests/CentOs7/Dockerfile

```diff
@@ -5,5 +5,5 @@ RUN yum install -y \
     https://download.docker.com/linux/centos/7/x86_64/stable/Packages/docker-ce-17.06.0.ce-1.el7.centos.x86_64.rpm && \
     yum clean all
 
-COPY run_ci_test_within_docker.sh /run_ci_test_within_docker.sh
-ENTRYPOINT ["/run_ci_test_within_docker.sh"]
\ No newline at end of file
+COPY run_ci_test_within_docker_bash_4.4-.sh /run_ci_test_within_docker_bash_4.4-.sh
```

Comment:
Is the '-' in run_ci_test_within_docker_bash_4.4-.sh intended?

Reply:
yes, we have + for everything above and 4.4 included, and 4.4- for everything below